app/uk/gov/hmrc/hec/controllers/TaxCheckController.scala

```diff
@@ -70,7 +70,8 @@ class TaxCheckController @Inject() (
             result => Ok(Json.toJson(result))
           )
 
-      case JsError(_) =>
+      case JsError(e) =>
+        logger.warn(s"error is $e")
```

Comment:
we should take this out I think - if this was in prod we could accidentally log sensitive info

Reply:
Yup, i added for testing my json parsing failure, forgot to remove it . Will do.

Reply:
done